UniWatch/Enumerations/EmailContentType.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace UniWatch.Enumerations
+{
+    public class EmailContentType
+    {
+        public string Value { get; private set; }
+
+        public static EmailContentType Plain => new EmailContentType("text/plain");
```

Comment:
Consider making these static fields in EmailService as opposed to a separate class?